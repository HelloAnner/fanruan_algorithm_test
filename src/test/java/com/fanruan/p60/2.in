5 6 4
7726640 9015357 8444123 5453453 9526765 639175 
2408343 1185523 8425225 756919 5137413 1685424 
4773738 9146479 2841788 2460729 5332217 5827975 
2651569 5766335 7194105 6126716 8029867 9408254 
3282465 8297337 8097870 4250022 5573379 1992768 
