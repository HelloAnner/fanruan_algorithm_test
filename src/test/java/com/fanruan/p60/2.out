8501160 8882219 8929764 9310823 